robot_skills/src/robot_skills/api.py

```diff
@@ -50,6 +56,26 @@ def query(self, description, grammar, target, timeout=10):
 
         return answer
 
+    def show_image(self, path_to_image, seconds=5.0):
+        """
+        Show an image on the HMI display interface
+        :param path_to_image: Absolute path to image file
+        :param seconds: How many seconds you would like to display the image on the screen
+        """
+        compressed_image_msg = CvBridge().cv2_to_compressed_imgmsg(cv2.imread(path_to_image))
```

Comment:
Is the creation of a new CvBridge instance each time the most efficient way?

Reply:
Certainly not, but given the frequency I don't consider it an issue. Would be nice, but there's more important stuff